lib/tmpl.js

```diff
@@ -186,4 +186,7 @@ var tmpl = (function() {
                   r.global ? 'g' : '')
   }
 
-})()
\ No newline at end of file
+})()
+
+if (typeof exports === 'object')
+  module.exports = tmpl
```

Comment:
The answer is here: https://github.com/muut/riotjs/pull/393#issuecomment-74825749